packages/material-icon/index.tsx

```diff
@@ -19,86 +19,61 @@
 // LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
+import * as React from 'react';
+import * as classnames from 'classnames';
+// https://github.com/material-components/material-components-web-react/issues/528
+// @ts-ignore
+import withRipple from '@material/react-ripple';
 
-import React from 'react';
-import classnames from 'classnames';
-import PropTypes from 'prop-types';
+export interface MaterialIconBaseProps {
+  icon?: string;
+  hasRipple?: boolean;
+  unbounded?: boolean;
+};
 
-import withRipple from '@material/react-ripple';
+export type MaterialIconProps = MaterialIconBaseProps & React.HTMLProps<HTMLElement>;
 
-export default class MaterialIcon extends React.Component {
-  render() {
-    const {
-      icon,
-      hasRipple,
-      ...otherProps
-    } = this.props;
+export default class MaterialIcon extends React.Component<
+  MaterialIconProps,
+  {}
+  > {
```

Comment:
This seems like it is formatted a bit oddly to me, was confused about the '<' on this line.

Reply:
Yeah I guess I'm not totally sure what would be better, separate lines are probably fine. Up to you!